README.md

```diff
@@ -8,18 +8,24 @@ This package serves as the entry point to the Formik Form state management libra
 
 Check our [Fonk Documentation site](https://lemoncode.github.io/fonk-doc/) and [Formik](https://lemoncode.github.io/fonk-doc/formik) section.
 
-How to install:
+Adding **Fonk** to **Formik** allows you to seamlessly add validation capabilities to **Formik**.
+
+In order to use **Fonk** with **Formik** you will have to install **fonk-formik** adaptor:
 
 ```bash
-npm install @lemoncode/fonk-formik --save
+npm install @lemoncode/fonk @lemoncode/fonk-formik--save
```

Comment:
Add white space between `@lemoncode/fonk-formik` and `--save`

Reply:
done